### ISSUE TITLE
undefined method 'cast_value' for nil:NilClass

### ISSUE THREAD

**Issue by archonic** [ankane/blazer#366]
This is happening for just previewing a table. The same statement is working in TablePlus. It's just `select * from "table_name" limit 10`. Blazer version is 2.2.1.

**Comment by ankane** [comment]
Hey @archonic, only the latest version of Blazer is supported, so if you're seeing it there, please paste the stack trace.

**Comment by archonic** [comment]
I tried upgrading and I've run into conflicting migration issues. The changelog doesn't have anything about upgrading. I guess the differences have to reconciled manually?

**Comment by ankane** [comment]
There shouldn't been any breaking changes with migrations between minor versions, so think it may be specific to your application.

**Comment by archonic** [comment]
I had rerun the install command thinking that might remedy things and noticed the differences. The error I saw was unrelated to the migration differences:

```
ArgumentError: wrong number of arguments (given 0, expected 1)
File "/app/app/views/blazer/dashboards/show.html.erb", line 13 in _app_views_blazer_dashboards_show_html_erb__3550120209871333095_2329220
```

I had to copy over the view files again and now it's working. I'm not actually overriding any blazer views - I should be able to just delete that folder and let it fallback to the gem views, right?
